rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,602 @@
+
+"""RDKit Conformer Browser
+Derived by Malitha Humayun Kabir from Greg Landrum's code as a part of GSoC 2017
+Project : RDKit - 3Dmol.js integration
+Mentors: Paul Czodrowski and Greg Landrum
+Acknowledgement: Peter Gedeck's suggestions helped to improve many lines of codes
+Date: 31th July 2017
+Email# malitha12345@gmail.com
+"""
+
+import py3Dmol
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from rdkit.Chem import Descriptors
+from ipywidgets import Layout, Label, Button, Box, HBox, VBox
+from ipywidgets import Dropdown, SelectMultiple, IntSlider, HTML, Checkbox, Button, Text
+from IPython.display import display
+import time
+
+
+##### Visualization
+
+BGCOLORS_3D = ('0x000000', '0xeeeeee', '0xffffff')
+
+PROP_RDKIT = tuple(sorted(prop for prop, _ in Descriptors._descList))
+
+DRAWING_LIGAND_3D=('line', 'cross', 'stick', 'cartoon', 'sphere', 'surface', 'ballstick')
+
+DRAWING_PROTEIN_3D=('line', 'cartoon', 'surface')
+
+COLOR_SCHEME_3D=('default', 'greenCarbon', 'cyanCarbon', 'magentaCarbon', 
+                 'yellowCarbon', 'whiteCarbon', 'orangeCarbon', 'purpleCarbon', 
+                 'blueCarbon', 'ssPyMOL', 'ssJmol', 'Jmol', 'amino', 
+                 'shapely', 'nucleic', 'chain', 'chainHetatm', 'prop')
+
+
+# I think this function is not required
+# User should be able to supply appropriate input
+# A notebook showing 3Dmol visualization should be present in rdkit doc
+def ProcessMolContainingObj(mol):
```

Comment:
The next commit would bring that function back since you are in favor of this function. 
The following is just for the clarification why I did that:
I was confused. I wrote that function when I wanted flexible input for testing. 
But my point is : the viewer needs a molecule id (this term is confusing though) that is not present in list object. so adding something new to the user supplied data may be confusing. Therefore, I have removed that dependency.